src/render-delaying-extensions.js

```diff
@@ -64,6 +65,7 @@ export function waitForExtensions(win) {
  */
 export function includedExtensions(win) {
   const document = win.document;
+  dev.assert(document.body);
```

Comment:
Seems very useful. The good thing about `dev.assert` is that one can have many of them without guilt.